example/sriov-network/sriov-network.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2021 NVIDIA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: example-sriov-network
+spec:
+  ipam: |
+    {
+      "type": "whereabouts",
+      "datastore": "kubernetes",
+      "kubernetes": {
+        "kubeconfig": "/etc/cni/net.d/whereabouts.d/whereabouts.kubeconfig"
+      },
+      "range": "192.168.4.225/28",
+      "exclude": [
+       "192.168.4.229/30",
+       "192.168.4.236/32"
+      ],
+      "log_file" : "/var/log/whereabouts.log",
+      "log_level" : "info"
+    }
+  vlan: 0
+  resourceName: nvidia.com/sriov-switchdev
```

Comment:
this will work for legacy. switchdev you will need to configure cni manauly like https://github.com/kubevirt/ovs-cni

Reply:
network-operator doesn't deploy ovs-cni. Should we mention it somewhere in this example?

Reply:
just add a comment that you need to manual configure the cni in switchev case